Offices/Offices.Tests/DoctorServiceTests.cs

```diff
@@ -0,0 +1,314 @@
+using AutoFixture;
+using AutoMapper;
+using MongoDB.Driver.Core.Misc;
+using Moq;
+using Offices.Application.MappingProfiles;
+using Offices.Application.Models;
+using Offices.Application.Services;
+using Offices.Data.Entities;
+using Offices.Data.Repositories.Abstractions;
+
+namespace Offices.Tests
+{
+    public class DoctorServiceTests
+    {
+        private readonly IMapper _mapper;
+
+        public DoctorServiceTests()
+        {
+            var configuration = new MapperConfiguration(cfg =>
+            {
+                cfg.AddProfile<ApplicationMappingProfile>();
+            });
+
+            _mapper = configuration.CreateMapper();
+        }
+
+        [Fact]
+        public async void GetDoctorById_Returns_Doctor()
+        {
+            // Arrange
+            var fixture = new Fixture();
+            var doctorRepositoryMock = new Mock<IDoctorRepository>();
+            doctorRepositoryMock.Setup(repo => repo.GetAsync(It.IsAny<string>(), CancellationToken.None))
+                              .ReturnsAsync(new Doctor
+                              {
+                                  Id = "1231231-1231-123",
+                                  FirstName = "John",
+                                  LastName = fixture.Create<string>(),
+                                  MiddleName = fixture.Create<string>(),
+                                  OfficeId = fixture.Create<string>(),
+                                  Status = fixture.Create<string>()
+                              });
+
+            var officeRepositoryMock = new Mock<IOfficeRepository>();
```

Comment:
This and setting up doctor service should be done in the constructor

Reply:
Fixed